cucumber-core/src/main/java/io/cucumber/core/eventbus/DefaultUuidGenerator.java

```diff
@@ -0,0 +1,14 @@
+package io.cucumber.core.eventbus;
+
+import java.util.UUID;
+
+/**
+ * Default UUID generator. The generator is thread-safe and supports multi-jvm
+ * usage of Cucumber.
+ */
+public class DefaultUuidGenerator implements UuidGenerator {
```

Comment:
Please rename this to `RandomUuidGenerator`. Should we ever switch to the fast generator as a default we won't get stuck on that.